packages/data/src/namespace-store/index.js

```diff
@@ -118,7 +118,7 @@ function createReduxStore( key, options, registry ) {
 		promise,
 	];
 
-	if ( options.controls ) {
+	if ( options.controls || options.resolvers ) {
 		const normalizedControls = mapValues( options.controls, ( control ) => {
 			return control.isRegistryControl ? control( registry ) : control;
 		} );
```

Comment:
Can `redux-routine` handle a potentially empty `normalizedControls`.

Reply:
Yes it can: https://github.com/WordPress/gutenberg/blob/9030e5198cee60e7c0e49fe919b7fb2e0dffabdc/packages/redux-routine/src/index.js#L19